modules/liberty/src/test/java/org/testcontainers/liberty/app/TestResource.java

```diff
@@ -0,0 +1,37 @@
+package org.testcontainers.liberty.app;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
```

Comment:
Tests failed because of the Java version. Currently, Testcontainers uses Java 8 but we can perform tests with other version declaring the following in the build.gradle

https://github.com/testcontainers/testcontainers-java/blob/8465795ab9067447ca8b7d3e307cadcec8de377a/modules/oracle-xe/build.gradle#L21-L32